www-apps/prewikka/prewikka-3.0.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 python-r1
+
+DESCRIPTION="Graphical front-end analysis console for the Prelude Framework"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="dev-python/setuptools
+	dev-python/lesscpy
+	${PYTHON_DEPS}"
+
+RDEPEND="dev-python/cheetah[$(python_gen_useflags python2_7)]
```

Comment:
[${PYTHON_USEDEP](https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#PYTHON_USEDEP) should be sufficient here